src/client/TargetsServerClient.js

```diff
@@ -14,39 +14,46 @@ class TargetClient {
     });
   }
 
+  static getMyTargets() {
+    return new Promise((resolve, reject) => {
+      instance.get('/targets').then((({ data })=> {
+        resolve(data);
+      })).catch(error => {
+        reject(error.message);
+      });
+    });
+  }
+
   static createTarget(target) {
-    return new Promise((resolve, reject)=> {
-      instance.post('/targets', target).then((res=> {
-        const {data} = res;
+    return new Promise((resolve, reject) => {
```

Comment:
Why did you wrap this into a Promise instance? 
Keep in mind that you instance.post returns a promise so you can simply do:

```
static updateTarget(target) {
    return  instance.post('/targets', target);
   }

updateTarget(...).then(success, failure)
 
```